docs/Playing/how-to-play.rst

```diff
@@ -1,224 +1,366 @@
-    contact with another player, don't worry about building military units just yet.
-
-All this time, you have been gaining techs as fast as possible. What you should be shooting for is first
-:strong:`The Republic`, then :strong:`Democracy`, then :strong:`Railroad`, and then
-:strong:`Industrialization`. Some people go for :strong:`Monarchy` before :strong:`The Republic`.  As soon
-as you've researched a new government type, start a revolution and change over to it. Cities operate much
-better as Republics than they do under Despotism, but note that it's much harder to keep military units
-outside of city limits under a Republic. Also, don't forget to recheck your rates after you've changed
+First of all, this isn't a *perfect* strategy, but it will get you started playing Freeciv21. Part of the
+appeal of Freeciv21 is in developing new strategies as you play more games. This guide is mostly based upon
+the Classic ruleset. There are other rulesets out there that ship with the game, but the base information
+presented here is the same for all rulesets.
+
+The game is divided into several steps or stages:
+
+#. Choosing the first city site.
+#. Selecting your first city locations.
+#. Selecting your technological advance.
+#. Mapping the countryside.
+#. Defending the cities.
+#. Deciding which units to build first.
+#. Improving the land.
+#. Deciding where to build cities.
+#. Taking care of the cities.
+#. Interacting with other players (diplomacy).
+#. Exploring the world.
+#. Things to keep in mind.
+#. Production phase.
+#. Annihilation of your opponents.
+
+Choosing the First City Site
+----------------------------
+
+Start the game by wandering around :strong:`briefly` to find a good place to build the first city. Do not feel
+tempted to investigate any yellow roofed villages yet, they might contain :unit:`Barbarian` tribes. Build the
+city close to Resources (fish, wheat, wine, or buffalo) and perhaps close to fresh water (River or Lake). Keep
+in mind that the first city you build will be your capital! It is advised not to place your capital on the
+coast. That can open you up to losing it from an attack from the sea.
+
+The idea is to balance the quality of the site you find against getting your first city established as early
+as possible. You will find that *balance* becomes a key aspect of playing Freeciv21.
+
+Selecting your First City Locations
+-----------------------------------
+
+In the Classic ruleset, you start the game with 5 :unit:`Settlers`, 6 :unit:`Workers`, and 2
+:unit:`Explorers`. Once you have planted your first city as your capital, you need to find a place for your
+remaining :unit:`Settlers` as quickly as possible. This initial expansion phase of the game is important to
+establishing your base cities. Cities are everything in Freeciv21. The more cities you have the more of every
+type of resource you can produce. Winning in Freeciv21 is a factor of your nation's ability to out produce
```

Comment:
```suggestion
type of resource you can produce. Winning in Freeciv21 is a factor of your nation's ability to out-produce
```

Could also be "outproduce" as one word, but that seems less clear to me than having it hyphenated

Reply:
Yea, I like the hyphenated version better too.